### DISCUSSION TITLE
Share and confirm the storage layout of your campus 🤝

### DISCUSSION THREAD

**Discussion by itislu** [itislu/42free#5] [Campuses]
### Here's a template you can use: <p align="right">▼&emsp;</p>

```
|     Campus      |       home        |       sgoinfre        |      OS       |     42free worked?     |
|       ---       |        ---        |          ---          |      ---      |          ---           |
|  <YOUR_CAMPUS>  |  <HOME_LIMIT> GB  |  <SGOINFRE_LIMIT> GB  |  Linux|macOS  |  Yes|No|Haven't tried  |
```
> [!NOTE]
> The `SGOINFRE_LIMIT` should be the limit per student, not the size of the whole storage device.

- If you confirm that 42free worked on your campus, I can add it to the [supported campuses 🌍](https://github.com/itislu/42free?tab=readme-ov-file#-confirmed-to-work-for-the-following-campuses).
- If the full template is filled, I can add your campus to the pre-configured setups users can choose from when installing 42free.

Thank you 😊

---

### Collected so far

|    Campus     |      home       |      sgoinfre       |       OS       |  42free worked?  |
|      ---      |       ---       |         ---         |      ---       |      ---       |
| 🇲🇦 1337 MED | 5 + 5 GB (₳$) | - | ??? | ??? |
| 🇧🇪 19 Antwerp | 5 GB | - | ??? | ??? |
| 🇧🇪 19 Brussels | 5 GB | 15 GB | macOS | Yes |
| 🇹🇭 42 Bangkok | 10 + 10 GB (A$) | 15 GB | ??? | Yes |
| 🇪🇸 42 Barcelona | 5 GB | 30 GB | ??? | ??? |
| 🇪🇸 42 Madrid | MIGRATING | MIGRATING | ??? | ??? |
| 🇪🇸 42 Malaga | 5 GB | 30 GB | ??? | ??? |
| 🇩🇪 42 Berlin | 5 GB | 30 GB | Linux | Yes |
| 🇩🇪 42 Heilbronn | 10 GB | - | ??? | ??? |
| 🇩🇪 42 Wolfsburg | 10 GB | 30 GB | ??? | ??? |
| 🇮🇹 42 Firenze | 5 GB | ??? GB | ??? | ??? |
| 🇰🇷 42 Gyeongsan | 5 GB | 0 + 10 GB (₳$) | Linux | Yes |
| 🇰🇷 42 Seoul | 5 GB | 0??? GB | ??? | ??? |
| 🇨🇭 42 Lausanne | 5  + 5 GB (₳$) | ∞??? GB | ??? | ??? |
| 🇵🇹 42 Lisboa | 5 GB | 30 GB | Linux | Yes |
| 🇬🇧 42 London | 5 + ??? GB (₳$) | ??? GB | ??? | ??? |
| 🇫🇷 42 Nice | 10 + 5 GB (₳$) | ∞ GB | ??? | ??? |
| 🇫🇷 42 Paris | 5 GB | 30 GB | ??? | ??? |
| 🇨🇿 42 Prague | 10 GB | ∞??? GB | ??? | ??? |
| 🇦🇹 42 Vienna | 10 GB | 30 GB | Linux | Yes |
| 🇵🇱 42 Warsaw | 10 GB | - | ??? | ??? |
| 🇳🇱 Codam | 10 GB | ∞ GB | ??? | ??? |

**Comment by Jonstep101010** [comment]
|   Campus    |     home      |     sgoinfre      |
|     ---     |      ---      |        ---        |
| Heilbronn | 10 | no Access |

**Reply by kurz-m** [reply]
We currently have no access to sgoinfre at 42 Heilbronn. So only 10gb of /home storage and then the local goinfre storage.

**Reply by Jonstep101010** [reply]
> We currently have no access to sgoinfre at 42 Heilbronn. So only 10gb of /home storage and then the local goinfre storage. 

Amended previous comment, thanks for pointing it out.

**Comment by Liquid-Core** [comment]
|      Campus       |     home      |     sgoinfre      |
|            ---            |      ---            |        ---              |
| S19 (belgium)  |       5 GB      |         15 GB      |

**Reply by itislu** [reply]
Thank you for the information!
Does this apply to both Antwerp and Brussels?

**UPDATE:**
This is for Brussels.

**Comment by Suigetsu** [comment]
|   Campus    |     home      |     sgoinfre      |
|     ---     |      ---      |        ---        |
|  1337 MED | 5 GB | no access |

home storage can be upgraded to 10 GB using Altarian Dollars. We don't have access to sgoinfre, but we can use the local goinfre storage

**Comment by San-tito** [comment]
|   Campus    |     home      |     sgoinfre      |
|     ---     |      ---      |        ---        |
| Barcelona | 5 GB | 30GB |

> Previously, you could expand your home storage to 10 GB. However, this feature has been temporarily disabled due to our ongoing migration to Linux throughout Spain. Consequently, we are uncertain about the current status of the storage allocation.

**Comment by wonhyeongseo** [comment]
|   Campus    |     home      |     sgoinfre      |
|     ---     |      ---      |        ---        |
| Gyeongsan | 5 GB | No Access |

**Comment by kcp2p** [comment]
|   Campus    |     home      |     sgoinfre      |
|     ---     |      ---      |        ---        |
| 🇹🇭 42 Bangkok | 5 GB (Pisciner) + 10 GB (Cadet) + 20 GB (A$) | 15 GB |

> Verified and confirmed by me.

> At 42 Bangkok, pisciner started at 5 GB and can be later upgraded to 10 GB if they decided to kick-off. Students can also purchase a storage upgrade to 20 GB using the Altairians Dollar.

> SGoinfre has a similar but soft limitations of 15 GB.

> OS is Linux

**Reply by itislu** [reply]
Thank you so much for the perfect report! I will add 42 Bangkok as a predefined set-up when installing 42free.

**Comment by hijae** [comment]
|     Campus      |       home        |       sgoinfre        |      OS       |     42free worked?     |
|       ---       |        ---        |          ---          |      ---      |          ---           |
|  Gyeongsan  |  5 GB  |  10 GB(A$)  |  Linux  |  Yes  |

**Reply by itislu** [reply]
Thank you for the report!
Do I understand correctly that the size of sgoinfre is 10 GB by default, but can be upgraded with A$? 
By how much can it be upgraded?

**Reply by hijae** [reply]
> Thank you for the report! Do I understand correctly that the size of sgoinfre is 10 GB by default, but can be upgraded with A$? By how much can it be upgraded?

Zero GB by default.

**Reply by itislu** [reply]
I see.

How does it work for a student who has **not** purchased the sgoinfre upgrade?
Do they have a personal directory but it gets cleaned every day/month, or they cannot even add anything into it?
Or they just don't have a personal directory at all until they buy the sgoinfre upgrade?

**Reply by hijae** [reply]
Home directories are personal directories that are allocated 5GB each, So sgoinfre is an optional upgrade.

**Reply by itislu** [reply]
I apologize, I should have been more clear.
So the 10 GB directory in sgoinfre only gets created in the file system when the student buys the upgrade?

**Reply by hijae** [reply]
I checked with the df command and everyone is visible. If I try to create a file without purchasing the upgrade, I get "disc quota exceeded"

**Reply by itislu** [reply]
Ok that's super useful information, thanks a lot!
I will add 42 Gyeongsan as a predefined set-up when installing 42free. For now the default will be 0 GB for sgoinfre, as this is the worst case. I might add a prompt to ask if the student has upgraded or not.
In all cases, the student can always change it manually with `42free --max-size`.

**Comment by zoeleu** [comment]
|     Campus      |       home        |       sgoinfre        |      goinfre       |     OS     |
|       ---       |        ---        |          ---          |      ---      |          ---           |
|  São Paulo  |  5 GB  |  4 TB  |  170 GB  |  Linux  |

Piscine 2026!

**Reply by itislu** [reply]
Thanks for the report!
Atm I only add set-ups for regular students, but if you get accepted I would be more than happy to add a set-up for 42 São Paulo with your help :)

For that I'd need to know:
1. Is there a limit per student for the sgoinfre?
2. Do the same home and sgoinfre limits apply to regular students?
3. Did 42free work flawlessly when trying it out on campus?

**Reply by zoeleu** [reply]
1. No clue
2. No clue
3. Didn't test

I verified the space available with `df -h`.
